data/scripts.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const {PRNG} = require('./../prng');
```

Comment:
You forgot to change this when you changed `prng`'s signature.

This is definitely a crash that TypeScript would have caught. :p